packages/text-bitmap/src/BitmapText.js

```diff
@@ -16,8 +16,9 @@ import { removeItems, getResolutionOfUrl } from '@pixi/utils';
  * let bitmapText = new PIXI.BitmapText("text using a fancy font!", {font: "35px Desyrel", align: "right"});
  * ```
  *
- * http://www.angelcode.com/products/bmfont/ for windows or
- * http://www.bmglyph.com/ for mac.
+ * http://www.angelcode.com/products/bmfont/ for Windows or
+ *
+ * http://www.bmglyph.com/ for Mac.
```

Comment:
I'd also add http://kvazars.com/littera/ as a free flash based online tool.

Reply:
Yeah, that's a nice tool, I was thinking of adding that, but then I thought it may not work on most of the browsers today without Flash. I hope they repackage it as a desktop app with Adobe AIR and distribute it as a standalone app.
There is another free tool called **Shoebox (Adobe AIR)**, runs as a standalone app on desktops. That might be a good addition too. (https://renderhjs.net/shoebox/)